internal/contour/listener.go

```diff
@@ -310,14 +313,14 @@ func httpfilter(routename string) *v2.Filter {
 	}
 }
 
-func filterchain(useproxy bool, filters ...*v2.Filter) *v2.FilterChain {
-	fc := v2.FilterChain{
+func filterchain(useproxy bool, filters ...envoy_api_v2_listener.Filter) envoy_api_v2_listener.FilterChain {
```

Comment:
Can this return a `*envoy_api_v2_listener.FilterChain`

Reply:
I see that the struct Listener type has changed it's struct field "Address" from
```go
Address *Address `protobuf:"bytes,2,opt,name=address" json:"address,omitempty"`
```
to
```go
Address envoy_api_v2_core.Address `protobuf:"bytes,2,opt,name=address" json:"address"`
```.

Is there a better way to convert the slice of pointers that `contour` is using to the slice of values that `envoy`'s APIs are expecting?